app/provider_details/rest.py

```diff
@@ -23,6 +24,14 @@ def get_providers():
     return jsonify(provider_details=data)
 
 
+@provider_details.route('/sms-statistics', methods=['GET'])
```

Comment:
Does this need to be a separate API call? Or can we update /provider-details to include the volume percentage?

Reply:
Using the existing api call is better - I've updated this PR and my admin one so we're not adding a new endpoint